front/app/src/Component/MenuBar/IconTitle.tsx

```diff
@@ -4,7 +4,7 @@ import	displayStyle from "./config/DisplayStyle";
 import	fontConfig from "./config/FontConfig";
 
 import { Icon } from "@mui/material";
-import icon from "./assets/icon_pure_white.svg";
+import icon from "./assets/icon.svg";
 
```

Comment:
fontConfig and icon are default import from their file (they could be named otherwise, they would work as the default export you can find in their file)
Icon is not default, it will work as itself (The export has the same name)
Sorry it's hard for me to explain anything at all. But it is not a mistake ^^